arch/x86/kvm/pkvm/vmx/host_vmentry.S

```diff
@@ -113,6 +113,8 @@ SYM_FUNC_START(pkvm_host_vmexit_entry)
 	mov VCPU_R15(%_ASM_BX), %r15
 	mov VCPU_RBX(%_ASM_BX), %_ASM_BX
 
+	CLEAR_CPU_BUFFERS
```

Comment:
nit: Commit message says "This will only effect the CPU which has the RFDS vulnerability" (BTW s/effect the CPU/affect a CPU/ :)) but AFAICS it is applied in case of some other vulnerabilities as well, e.g. MDS? I understand we don't care about those vulnerabilities for now, but still, the statement is not quite accurate (given that we have relax_cpu_bugs so pKVM still can run on older CPUs as well)?

Reply:
Makes sense. How about change the commit as below:
"
    Clear CPU buffers before entering the host VM from the pKVM hypervisor.
    This will only affect CPUs which have feature X86_FEATURE_CLEAR_CPU_BUF
    to mitigate the CPU vulnerabilities, e.g., RFDS, MDS, TAA. If a CPU
    doesn't have such vulnerability, this feature bit will not be set thus
    it will do nothing.
"